proof/drefine/Arch_DR.thy

```diff
@@ -1082,11 +1082,6 @@ lemma pde_opt_cap_eq:
   apply (clarsimp simp: valid_idle_def st_tcb_at_def obj_at_def pred_tcb_at_def)
   done
 
-lemma corres_add_noop_rhs:
```

Comment:
you're modifying drefine but I didn't see any `drefine` tag for any of the commits

Reply:
ah, I think I see, there's a commit that says `add ... to ...`, which is confusing. It should at minimum say `move ... to ...`, but I'd prefer adding the tags as well